app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -62,6 +62,19 @@ $(function() {
           width: 'resolve',
           allowClear: allowClear
         });
+
+        var $parent = $(el).parent(), $label = $parent.find('label');
+
+        if ($label.length) {
+          var $input = $parent.find('#' + $label.attr('for'))
+
+          if (!$input.attr('id') == $(el).attr('id')) {
+            $(el).on('change', function(value) {
+              $input.attr('name', 'q[' + $(this).val() + ']')
```

Comment:
Unexpected string concatenation                                    prefer-template<br>'$' is not defined                                                 no-undef<br>Missing semicolon                                                  semi